### PR TITLE
`destination-bigquery` - better config groups

### PR DESCRIPTION
**Pull request by evantahler**
Updates some of the SPEC descriptions and groups to (1) simplify the UI and (2) call out that staging=good, standard-inserts=bad

BEFORE:
Screenshot 2023-10-09 at 4 01 07 PM

AFTER:
Screenshot 2023-10-09 at 4 01 00 PM

### CODE REVIEW

File: airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json [hunk 1]
```diff
@@ -172,6 +178,7 @@
           {
             "title": "Standard Inserts",
             "required": ["method"],
+            "description": "<i>(not recommended)</i> Direct uploading using SQL INSERT statements. This method is extremely inefficient and provided only for quick testing. In almost all cases, you should use staging.",
```
Comment (Hesperide) [inline airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json:181]:
```suggestion
            "description": "<i>(not recommended)</i> Direct uploads using SQL INSERT statements. This method is extremely inefficient and provided only for quick testing. In all other cases, you should use GCS staging.",
```

Comment (Hesperide) [reply airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json:181]: Slightly stronger wording

File: airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json [hunk 2]
```diff
@@ -69,16 +71,20 @@
...
+        "description": "Loading method used to send select the way data will be uploaded to BigQuery.",
+        "display_type": "radio",
+        "group": "connection",
         "order": 3,
         "oneOf": [
           {
             "title": "GCS Staging",
+            "description": "<i>(recommended)</i> Writes large batches of records to a file, uploads the file to GCS, then uses COPY INTO to upload the file. Recommended for most workloads for better speed and scalability. Read more about GCS Staging <a href=\"https://docs.airbyte.com/integrations/destinations/bigquery#gcs-staging\">here</a>.",
```
Comment (Hesperide) [inline airbyte-integrations/connectors/destination-bigquery/src/main/resources/spec.json:87]:
```suggestion
            "description": "<i>(recommended)</i> Writes large batches of records to a file, uploads the file to GCS, then uses COPY INTO to load your data into BigQuery. Provides best-in-class speed, reliability and scalability. Read more about GCS Staging <a href=\"https://docs.airbyte.com/integrations/destinations/bigquery#gcs-staging\">here</a>.",
```

**Review by Hesperide — APPROVED** [review]

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`0ec301083c`](https://github.com/airbytehq/airbyte/pull/31198/commits/0ec301083c7f039de4193293b4ca405336a13d82)) - ✅

⏲️  Total pipeline duration: 09mn09s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_standard-inserts-bad/1696892807/0ec301083c7f039de4193293b4ca405336a13d82/destination-bigquery/2.0.26/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:0ec301083c7f039de4193293b4ca405336a13d82)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`685f4e4312`](https://github.com/airbytehq/airbyte/pull/31198/commits/685f4e43120a66293f30b786cebf76dd0a545a1c)) - ✅

⏲️  Total pipeline duration: 08mn29s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_standard-inserts-bad/1696895080/685f4e43120a66293f30b786cebf76dd0a545a1c/destination-bigquery/2.0.26/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:685f4e43120a66293f30b786cebf76dd0a545a1c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`32dc375dbc`](https://github.com/airbytehq/airbyte/pull/31198/commits/32dc375dbc7155d7f0adb077abcaf9a334ecda12)) - ✅

⏲️  Total pipeline duration: 09mn03s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/evan_standard-inserts-bad/1696952751/32dc375dbc7155d7f0adb077abcaf9a334ecda12/destination-bigquery/2.0.26/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:32dc375dbc7155d7f0adb077abcaf9a334ecda12)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Review by edgao — APPROVED** [review]

**Pull request merged**